File: Anyelo Carrillo C++/Ley Ohm.cpp
#include <iostream>

int main() {
    int Voltaje, Resistencia;
    std::cout << "Ingrese el valor del voltaje del circuito: ";
    std::cin >> Voltaje;
    std::cout << "Ingrese el valor de la resistencia del circuito: ";
    std::cin >> Resistencia;

    double Intensidad = static_cast<double>(Voltaje) / Resistencia;
    std::cout << "Al conectar un resistor de R " << Resistencia << " ohm a una fuente de V " << Voltaje << " voltios, circulará una corriente de " << Intensidad << " amperios" << std::endl;

    return 0;
}

File: Anyelo Carrillo Java/Ley Ohm.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        // Crear un objeto Scanner para leer la entrada del usuario
        Scanner scanner = new Scanner(System.in);

        // Solicitar y leer el valor del voltaje
        System.out.print("Ingrese el valor del voltaje del circuito: ");
        double Voltaje = scanner.nextDouble();

        // Solicitar y leer el valor de la resistencia
        System.out.print("Ingrese el valor de la resistencia del circuito: ");
        double Resistencia = scanner.nextDouble();

        // Calcular la intensidad (corriente)
        double Intensidad = Voltaje / Resistencia;

        // Mostrar el resultado
        System.out.println("Al conectar un resistor de R " + Resistencia + " ohm a una fuente de V " + Voltaje + " voltios, circulará una corriente de " + Intensidad + " amperios");

        // Cerrar el Scanner
        scanner.close();
    }
}
